modules/document_repository/README.md

```diff
@@ -0,0 +1,46 @@
+# Document Repository
+
+## Purpose
+
+The Document Repository is a tool that provides a centralized location for non-candidate documents relevant to the study. For example, copies of non-filled instruments, publications, manual of operations.
```

Comment:
suggestion: "The Document Repository is a tool that provides a centralized location for non-candidate documents that are relevant to the study i.e. copies of unfilled instruments, publications, and manual of operations."

Reply:
I think "unfilled versions of instruments" would be better than "copies of unfilled instruments"

Reply:
and maybe "and/or" instead of "and" (or even just "or")